File: Code coach/Easy/Hovercraft/hovercraft.cpp
#include <iostream>
using namespace std;

int main()
{
	int unitsSold;
	cin >> unitsSold;

	int sellPrice = 3000000;
	int costPrice = 2000000;
	int insurancePrice = 1000000;

	int unitsBuilt = 10;
	int expenses = (unitsBuilt * costPrice) + insurancePrice;
	int benefit = unitsSold * sellPrice;

	if (benefit > expenses)
	{
		cout << "Profit";
	}
	else if (benefit < expenses)
	{
		cout << "Loss";
	}
	else
	{
		cout << "Broke Even";
	}

	return 0;
}

File: Code coach/Easy/Hovercraft/hovercraft.py
units_sold = int(input())

sell_price = 3_000_000
cost_price = 2_000_000
insurance_price = 1_000_000

units_built = 10
expenses = (units_built * cost_price) + insurance_price
benefit = units_sold * sell_price

if benefit > expenses:
	print("Profit")
elif benefit < expenses:
	print("Loss")
else:
	print("Broke Even")
